extensions/amp-quicktext/OWNERS

```diff
@@ -0,0 +1,16 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+  {
+    owners: [
+      {name: 'satoripop/amp'},
```

Comment:
Non-`ampproject` GitHub teams are not supported as owners at this time. All extensions should generally also have an `ampproject` working group owner, in this case probably `ampproject/wg-ui-and-a11y`